frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -1493,23 +1546,19 @@ export class Arena {
       }
     }
 
-    $('.anchor', r).attr('name', anchor);
-    $('.contest', r).text(clarification.contest_alias || '');
-    $('.problem', r).text(clarification.problem_alias);
+    r.querySelector('.anchor')?.setAttribute('name', anchor);
+    setItemText(r, '.contest', clarification.contest_alias ?? '');
+    setItemText(r, '.problem', clarification.problem_alias);
     if (this.problemsetAdmin) {
-      $('.author', r).text(clarification.author || '');
+      setItemText(r, '.author', clarification.author ?? '');
     }
-    $('.time', r).html(time.formatTimestamp(clarification.time));
-    $('.message', r).text(clarification.message);
-    $('.answer pre', r).text(clarification.answer ?? '');
+    setItemHtml(r, '.time', time.formatTimestamp(clarification.time));
```

Comment:
```suggestion
    setItemText(r, '.time', time.formatTimestamp(clarification.time));
```

esto ya no usa HTML!